From 4b3bda9331c02c2295087ab315f9b4d062f7e8d0 Mon Sep 17 00:00:00 2001
From: Reuben Kuhnert <reuben.kuhnert@cloudera.com>
Date: Mon, 23 Nov 2015 10:23:57 -0600
Subject: [PATCH 0419/1164] PROPOSED CDH-34567: HIVE-12469: Apache Commons
 Collections - HIVE

This patch bumps the version of commons collections from 3.2.1 to 3.2.2 to fix the InvokerTransformer security vulnerability.
See: https://issues.apache.org/jira/browse/COLLECTIONS-580 for more information.

Change-Id: I195b87cae4e47c6baf099a338305babdbb129850
---
 accumulo-handler/pom.xml |    6 ++++++
 pom.xml                  |    6 ++++++
 2 files changed, 12 insertions(+)

diff --git a/accumulo-handler/pom.xml b/accumulo-handler/pom.xml
index 7521b68..7c717f6 100644
--- a/accumulo-handler/pom.xml
+++ b/accumulo-handler/pom.xml
@@ -43,6 +43,12 @@
     <dependency>
       <groupId>org.apache.accumulo</groupId>
       <artifactId>accumulo-core</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>commons-collections</groupId>
+          <artifactId>commons-collections</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
     <dependency>
       <groupId>org.apache.accumulo</groupId>
diff --git a/pom.xml b/pom.xml
index 49909c7..177377f 100644
--- a/pom.xml
+++ b/pom.xml
@@ -110,6 +110,7 @@
     <findbugs.version>1.3.9</findbugs.version>
     <commons-cli.version>1.2</commons-cli.version>
     <commons-codec.version>1.4</commons-codec.version>
+    <commons-collections.version>3.2.2</commons-collections.version>
     <commons-compress.version>1.4.1</commons-compress.version>
     <commons-exec.version>1.1</commons-exec.version>
     <commons-httpclient.version>3.0.1</commons-httpclient.version>
@@ -283,6 +284,11 @@
         <version>${parquet.version}</version>
       </dependency>
       <dependency>
+        <groupId>commons-collections</groupId>
+        <artifactId>commons-collections</artifactId>
+        <version>${commons-collections.version}</version>
+      </dependency>
+      <dependency>
         <groupId>com.twitter</groupId>
         <artifactId>parquet-column</artifactId>
         <version>${parquet.version}</version>
-- 
1.7.9.5

